tools/build_langserver/langserver/fs.go

```diff
@@ -0,0 +1,64 @@
+package langserver
+
+import (
+	"context"
+	"encoding/json"
+
+	"tools/build_langserver/lsp"
+
+	"github.com/sourcegraph/jsonrpc2"
+)
+
+func (h *LsHandler) handleFSRequests(ctx context.Context, req *jsonrpc2.Request) (result interface{}, err error) {
```

Comment:
why are these "file system methods"? they all seem to be named "textDocument/something"